python-main.js

```diff
@@ -500,24 +500,40 @@ function web_editor(config) {
         $('.fs-file-list table tbody').empty();
         micropythonFs.ls().forEach(function(filename) {
             var pseudoUniqueId = Math.random().toString(36).substr(2, 9);
-            // Check for main.py to exclude it from the table list
-            if (filename === 'main.py') return;
-            var fileType = (/[.]/.exec(filename)) ? /[^.]+$/.exec(filename) : "";
+            var name = filename;
+            var disabled = "";
+            if (filename === 'main.py') {
+              name = getName() + " (" + filename + ")";
+              disabled = "disabled";
+            };
             $('.fs-file-list table tbody').append(
-                '<tr><td>' + filename + '</td>' +
-                '<td>' + fileType + '</td>' +
+                '<tr><td>' + name + '</td>' +
                 '<td>' + (micropythonFs.size(filename)/1024).toFixed(2) + ' Kb</td>' +
-                '<td><button id="' + pseudoUniqueId + '" class="fs-remove-button">Remove</button></td></tr>'
+                '<td><button id="' + pseudoUniqueId + '2" class="save-button ' + disabled + '" title="Remove" style="width: 4em; height: 2em; float:right;"><i class="fa fa-trash"></i></button>' +
```

Comment:
Rather than add a `2` after the id name, can we give it a more descriptive string like `_delete` or `_remove`?